### PR TITLE
Create `mutable and immutable` task

### PR DESCRIPTION
**Pull request by IvanRamyk**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+name = "Richard"
+my_favourite_films = [
+    "The Shawshank Redemption",
+    "The Lord of the Rings: The Return of the King",
+    "Pulp Fiction",
+    "The Good, the Bad and the Ugly",
+    "The Matrix",
+]
+profile_info = ("michel", "michel@gmail.com", "12345678")
+marks = {
```
Comment (danylott) [inline app/main.py:13]: You prefer not to add set?

File: tests/test_main.py [hunk 2]
```diff
@@ -1,6 +1,94 @@
...
+def test_variables_should_be_added_to_sorted_variables(variable_name):
+    sorted_variables = getattr(app.main, "sorted_variables")
+    assert (
+        getattr(app.main, variable_name) in sorted_variables["mutable"]
+        or getattr(app.main, variable_name) in sorted_variables["immutable"]
+    ), f"Variable '{variable_name}' should be added to 'sorted_variables'"
+
+
+def is_immutable(obj):
+    if (
```
Comment (danylott) [inline tests/test_main.py:73]: I know, that students don't know this, but still, you'd better use it:
https://stackoverflow.com/questions/33311258/python-check-if-isinstance-any-type-in-list

File: tests/test_main.py [hunk 3]
```diff
@@ -1,6 +1,94 @@
...
+    sorted_variables = getattr(app.main, "sorted_variables")
+    for variable in sorted_variables["mutable"]:
+        assert is_immutable(variable) is False, (
+            f"{variable} should be in 'immutable' list"
+        )
+
+    for variable in sorted_variables["immutable"]:
+        assert is_immutable(variable) is True, (
+            f"{variable} should be in 'mutable' list"
+        )
```
Comment (danylott) [inline tests/test_main.py:94]: No new line at the end of the file

Comment (danylott) [reply tests/test_main.py:94]: We need to check flake8 for our commits also.
Maybe it makes sense to change the order:
run flake8 first on GitHub workflow, and only after run tests

**Review by danylott — CHANGES_REQUESTED** [review]

File: tests/test_main.py (hunk 3 above)
Comment (IvanRamyk) [reply tests/test_main.py]: _No new line at the end of the file_: **Fixed**
_run flake8 first on GitHub workflow, and only after run tests_: **Updated**

File: tests/test_main.py (hunk 2 above)
Comment (IvanRamyk) [reply tests/test_main.py]: Thanks, applied :)

File: app/main.py (hunk 1 above)
Comment (IvanRamyk) [reply app/main.py:13]: set variable is added

**Review by danylott — APPROVED** [review]

**Pull request merged**